docs/src/getting-started/cli.md

```diff
@@ -107,7 +108,8 @@ export MIDEN_FAUCET_MAX_SUPPLY=
 
 # Faucet Service Configuration
 export MIDEN_FAUCET_FRONTEND_URL=http://localhost:8080
-export MIDEN_FAUCET_API_URL=http://localhost:8000
+export MIDEN_FAUCET_API_BIND_URL=http://localhost:8000
```

Comment:
Would be nice to tackle https://github.com/0xMiden/miden-faucet/issues/152#issuecomment-3518547757, unless we think there's some reason not to right now